src/components/WebexMeetingControl/WebexMeetingControl.jsx

```diff
@@ -27,21 +57,28 @@ function renderButton(action, display, cssClasses, style, showText) {
   } = display;
   const isDisabled = display.state === MeetingControlState.DISABLED;
   const iconColor = type === 'CANCEL' || display.state === MeetingControlState.ACTIVE ? 'red' : '';
+  const buttonType = getButtonType(type);
```

Comment:
this can be a constant map defined between the imports and the component function, and used like this:

```jsx
<Button type={controlType2ButtonType[type] || 'default'} ... />
```